packages/components/src/systems/PatientMedHistory/index.tsx

```diff
@@ -42,6 +43,25 @@ type PatientMedHistoryProps = {
   newMedication?: Medication | SearchMedication;
 };
 
+type PatientMedication = {
+  active: boolean;
+  prescription: {
```

Comment:
Schema says this is optional https://github.com/Photon-Health/terraform/blob/main/modules/graphql-api/appsync.graphql#L86